sys-apps/archspec/archspec-0.1.3.ebuild

```
@@ -4,23 +4,29 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{6..10} )
-
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
 inherit distutils-r1
 
-#DISTUTILS_USE_SETUPTOOLS=pyproject.toml
-
 DESCRIPTION="A library to query system architecture"
 HOMEPAGE="https://github.com/archspec/archspec"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+# Use the tarball from GitHub, as the one from PyPI does not contain the tests.
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+# The tarball from GitHub does not contain the archspec microarchitectures data file,
+# which is part of the archspec-json repository. So, we need to download that tarball as well.
+ARCHSPEC_JSON="${PN}-json.tar.gz"
```

Hmm, we should retain the version here too, no?

```suggestion
ARCHSPEC_JSON="${PN}-json-${PV}.tar.gz"
```

---

Well spotted! :-) 
Should definitely be there, indeed, so I just added it, and updated the Manifest.